qa/common/clara_deploy_pipelines.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+https://api.ngc.nvidia.com/v2/resources/nvidia/clara/clara_ai_pancreas_pipeline/versions/0.7.2-2010.1/zip
```

Comment:
I think it would be better to create our own CI models but this looks good too. For example, creating a model that uses a lot of memory or another model that uses a lot of compute, etc.